sections/04_study.md

```diff
@@ -640,8 +640,11 @@ variant and fed these vectors into a fully connected deep neural network
 [@tag:Torracinta2016_deep_snp]. Unfortunately, this feature set required at
 least 15 iterations of software development to fine-tune, which will likely not
 be generalizable. Going forward, we foresee that variant calling will benefit
-most from optimized neural network architectures and better methods of encoding
-raw sequence and quality data (e.g. in tensor rather than RGBA format).
+more from optimizing neural network architectures than from developing features
+by hand. An interesting and informative next step would be to test whether
```

Comment:
It seems silly to put so much emphasis on a single sentence, but what do you think about this:

"An interesting and informative next step would be to test whether encoding raw sequence and quality data as an image, tensor, or some other mixed format produces the best variant calls, given identical network architectures and other parameters.  The advantages of encoding genomic data as images in order to initialize weights from networks pre-trained on generic, large-scale image datasets may diminish over time or prove to be a generally effective strategy."

The phrasing of my new sentence is awkward though.